tests/publiccloud/create_aistack_env.pm

```diff
@@ -45,8 +49,9 @@ sub install_dependency_package {
 
     # rke2 activation section
     script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
-    script_run("sh ./install_rke2.sh");
+    script_run("sh ./install_rke2.sh | sh -");
```

Comment:
Why do you add here `sh -`? Script executed already in first `sh`, the pipe shouldn't be there.

Reply:
Removed